awx/main/tests/unit/models/test_ha.py

```diff
@@ -59,15 +61,17 @@ class TestInstanceGroup(object):
         ],
     )
     def test_fit_task_to_most_remaining_capacity_instance(self, task, instances, instance_fit_index, reason):
-        ig = InstanceGroup(id=10)
+        InstanceGroup(id=10)
+        tm_igs = TaskManagerInstanceGroups([])
 
-        instance_picked = ig.fit_task_to_most_remaining_capacity_instance(task, instances)
+        instance_picked = tm_igs.fit_task_to_most_remaining_capacity_instance(task, instances=instances)
 
         if instance_fit_index is None:
             assert instance_picked is None, reason
         else:
             assert instance_picked == instances[instance_fit_index], reason
 
+    @pytest.mark.django_db
```

Comment:
Anything marked with django_db should be in the awx/main/tests/unit folder. I just want to give you the general principal here. I'm generally find converting unit tests to functional tests as the need arises based on how things are written.